Codeinwp/neve
# Add Content Vertical Spacing control

### Summary
- Added a new control for Content Vertical Spacing.
- The control allows you to customize the content's top and bottom margins
Note: I also added support for CPT module, that's treated in a PR that will be made in Neve Por repo

### Will affect the visual aspect of the product
NO

### Screenshots 
https://vertis.d.pr/v/A9cpqv

### Test instructions
- Test the Global Vertical Spacing that applies for both pages and posts
- Save the Global Vertical Spacing, refresh the customizer page and go to Customizer -> Layout -> Single post -> Page settings. Move the control to "Custom" and check if the default value is the same as Global Vertical Spacing
- Test the Post Vertical Spacing
- Test the Page Vertical Spacing

## Check before Pull Request is ready:



Closes Codeinwp/neve-pro-addon#1802.
